CAP/gap/CategoryObjectsOperations.gi

```diff
@@ -132,6 +132,7 @@ InstallMethod( SetOfObjectsAsUnresolvableAttribute,
         
   SetOfObjectsOfCategory );
 
+#= comment for Julia
 ##
 InstallMethodForCompilerForCAP( SetOfObjects,
         [ IsCapCategory and HasOppositeCategory ],
```

Comment:
Would it work if you just delete `IsCapCategory and`?

Reply:
Using only HasOppositeCategory results raises the Error:

```
ERROR: LoadError: type #HasOppositeCategory has no field abstract_type
Stacktrace:
  [1] getproperty(x::Function, f::Symbol)

```

Reply:
I see, thanks.